pyvo/dal/__init__.py

```diff
@@ -1,9 +1,11 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from .sia import search as imagesearch
+from .sia2 import search as imagesearch2
 from .ssa import search as spectrumsearch
 from .sla import search as linesearch
 from .scs import search as conesearch
 from .tap import search as tablesearch
+from .sia2 import *
```

Comment:
I've said it on a different occasion, this is pretty much the only time I use the wildcard import and that is because we have a `__all__` defined in sia2, which limits the scope of the wild card to the
items in that list. If you have a strong opinion, I can replace the wild card with the content of the `__all__` list but that to me is just duplicate info that one has to remember to keep in sync.

Reply:
Ah that one. Well, got no strong opinion either, but to follow PEP20 there should be only one way to do it.

Reply:
I've removed the import all together. It was colliding with the `sia` import anyhow. Now one has to explicitly import from the `sia2` module, e.g. `pyvo.dal.SIAService` points to `sia` implementation and `pyvo.dal.sia2.SIAService` points to `sia2`.